src/pages/areas/[id].tsx

```diff
@@ -181,19 +181,26 @@ export const getStaticProps: GetStaticProps<AreaPageProps, {id: string}> = async
       }
     }
 
-    let mediaListWithUsernames = rs.data.area.media
+    let mediaListWithUsernames: MediaBaseTag[] = rs.data.area.media
     try {
       mediaListWithUsernames = await enhanceMediaListWithUsernames(rs.data.area.media)
     } catch (e) {
       console.log('Error when trying to add username to image data', e)
     }
 
+    /**
+     * Call Sirv API to get image metadata.  We should probably store metadata in the db.
+     */
+    const mediaListWithMetadata = await getImagesByFilenames(mediaListWithUsernames.map(entry => entry.mediaUrl))
+
+    const mediaMetaDict = indexBy<MediaType[]>(mediaListWithMetadata.mediaList, 'mediaId')
```

Comment:
@clintonlunn we should probably store image width & height in the DB in the future.